runtime/doc/dev_style.txt

```diff
@@ -1008,20 +914,8 @@ Use of horizontal whitespace depends on location.
         x = -5;           // No spaces separating unary operators and their
         x++;              // arguments.
         if (x && !y)
-          ...
-        i = (int)d;       // No spaces after a cast operator.
 <
 
-Vertical Whitespace ~
-
-Minimize use of vertical whitespace.
-
-The basic principle is: The more code that fits on one screen, the easier it
-is to follow and understand the control flow of the program. Of course,
-readability can suffer from code being too dense as well as too spread out, so
-use your judgment. But in general, minimize use of vertical whitespace.
```

Comment:
This is not always automatable, so worth keeping.

I sometimes see really egregious over-use of blank lines.

Reply:
It should be automated already. Uncrustify squeezes all consecutive newlines to a maximum of two. If not I need to write a issue report in their repo.

Reply:
It's not actually clear if `:help dev-stile` applies only C code. If we want to insist that it does, then we need a new section of "general guidelines" that apply to all code. This would be one of those guidelines.